content/500-platform/550-platfrom-cli/commands.mdx

```diff
@@ -0,0 +1,230 @@
+---
+title: 'Commands'
+metaTitle: 'Platform CLI: Commands'
+earlyaccess: true
+tocDepth: 5
+---
+
+<TopBlock>
+
+This document describes the Prisma Data Platform's integrated Prisma CLI commands, arguments, and options.
+
+</TopBlock>
+
+## Getting started
+
+To get started, ensure you have the [Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client) updated to version `5.7.0` or later. This is necessary to access the Platform through the Prisma CLI.
+
+<Admonition>
+
+💡 When using commands, always start with `prisma platform` and include the `--early-access` flag to enable the use of the Prisma Data Platform whilst still in early access.
+
+</Admonition>
+
+## Authentication
+
+### <inlinecode>platform</inlinecode>
+
+#### <inlinecode>auth login</inlinecode>
+
+Logs into your Prisma Data Platform account or creates a new one.
```

Comment:
Can we be more specific? This actually opens a browser window where you can log in, right? I think that should be mentioned, right now it sounds like the command magically logs you in or creates a new account.